engine/lite/kafka/runtime/src/main/scala/pl/touk/nussknacker/engine/lite/kafka/NuKafkaRuntimeApp.scala

```diff
@@ -75,7 +83,11 @@ object NuKafkaRuntimeApp extends App with LazyLogging {
     }
   }
 
-  private def prepareScenarioInterpreter(runtimeConfig: Config): KafkaTransactionalScenarioInterpreter = {
+  private def parseDeploymentData(path: Path): LiteKafkaJobData = {
```

Comment:
I think this is not a good name for this config, since we already have DeploymentData class (which we don't parse properly yet...)

Reply:
Adding prefix `Lite` will be ok for you?

Reply:
deploymentConfig?

Reply:
why config? configs are something that are configured by user, not code

Reply:
It's are data/parameters passed to invocation of runtime, and it is passed similar as `DeploymentData` in Flink, so IMO name should be similar. It will be strange, if someone will ask you, how it is called in flink, and you will say that it is some totally other name

Reply:
but DeploymentData is in deployment-api... for me it's a bit confusing to have two of them. Maybe we should additionalDeploymentData field or sth?

Reply:
After rethinking - it can be deployment config. will change